packages/helpwave_localization/lib/l10n/app_en.arb

```diff
@@ -1,4 +1,10 @@
 {
+  "station": "stations",
```

Comment:
"station" is only for German, the international term would be "ward" therefore the textId should also be ward